controllers/storagecluster/reconcile.go

```diff
@@ -144,6 +146,19 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 	defer func() { r.Log = prevLogger }()
 	r.Log = r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 
+	oc := &apiv1.OperatorCondition{}
+	err := r.Client.Get(ctx, types.NamespacedName{Name: "ocs-operator.v4.9.0", Namespace: request.Namespace}, oc)
```

Comment:
The ocs-operator version should not be hardcoded. This means it has to be changed with every release

Reply:
yes, I ll change that